cmd/agent/gui/views/templates/collectorStatus.tmpl

```diff
@@ -34,6 +34,7 @@
                 {{ $k }}: Last Run: {{humanize (index $instance.EventPlatformEvents $k) }}, Total: {{humanize $v}}<br>
                 {{- end -}}
                 Service Checks: {{humanize .ServiceChecks}}, Total: {{humanize .TotalServiceChecks}}<br>
+                Histogram Buckets: {{humanize .HistogramBuckets}}, Total: {{humanize .TotalHistogramBuckets}}<br>
```

Comment:
I think we want to display `Histogram Buckets` only when it is not zero as few checks expose this metric.

Reply:
Done in [92ab150](https://github.com/DataDog/datadog-agent/pull/11365/commits/92ab15081f332f000c751a78d017bfa51126ffd1)